pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3429,7 +3430,7 @@ type RadosNamespaceMirroring struct {
 	// RemoteNamespace is the name of the CephBlockPoolRadosNamespace on the secondary cluster CephBlockPool
 	// +optional
 	RemoteNamespace *string `json:"remoteNamespace"`
-	// Mode is the mirroring mode; either pool or image
+	// Mode is the mirroring mode; either pool or image. Refer to [namespace mirroring doc](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#namespace-configuration)
```

Comment:
The rook docs already have a link to this topic, let's not put a link here so it doesn't get copied so many places in the CRD generation.
```suggestion
	// Mode is the mirroring mode; either pool or image.
```

Reply:
done